### PR TITLE
feat(36, 20): added ui for Shelves List, Shelf, Flowers List and Flow…

### PR DESCRIPTION
**Pull request by karolinayanko**
Added ui for Shelves List, Shelf, Flowers List and Flower.
Added Font Awesome to the project.
Added basic routes

### CODE REVIEW

File: packages/web/src/components/Shelves/FlowerItemAdd/FlowerItemAdd.test.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { FlowerItemAdd } from '../FlowerItemAdd';
```
Comment (belgithub) [inline packages/web/src/components/Shelves/FlowerItemAdd/FlowerItemAdd.test.tsx]: Git suggest next changes:
Useless path segments for "../FlowerItemAdd", should be "./"

File: packages/web/src/components/Shelves/FlowerItem/FlowerItem.test.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { FlowerItem } from '../FlowerItem';
```
Comment (belgithub) [inline packages/web/src/components/Shelves/FlowerItem/FlowerItem.test.tsx]: Git suggest next changes:
Useless path segments for "../FlowerItem", should be "./"

**Review by dubbha — APPROVED** [review]

File: packages/web/src/components/Shelves/FlowerItemAdd/FlowerItemAdd.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+import { Card } from 'react-bootstrap';
+
+import '../styles.scss';
```
Comment (belgithub) [inline packages/web/src/components/Shelves/FlowerItemAdd/FlowerItemAdd.tsx:6]: Is it correct path and do we need it?

File: packages/web/src/components/Shelves/Shelf/Shelf.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+export const Shelf = ({
+  id
+}: Props) => {
+  // const {
+  //   isLoading,
+  //   error,
+  //   info
+  // } = useSelector(getShelf);
```
Comment (dartwlad) [inline packages/web/src/components/Shelves/Shelf/Shelf.tsx:17]: Don't forget to remove it.

**Review by dartwlad — APPROVED** [review]

File: packages/web/src/components/Shelves/Shelf/Shelf.tsx (hunk 4 above)
Comment (dubbha) [reply packages/web/src/components/Shelves/Shelf/Shelf.tsx:17]: We'll pass it through dev and add the actual logic via redux while merging with another feature

**Review by belgithub — APPROVED** [review]

**Pull request merged**